build.assets/makefiles/master/k8s-master/kube-controller-manager.service

```diff
@@ -16,7 +16,7 @@ ExecStart=/usr/bin/kube-controller-manager \
         --kubeconfig=/etc/kubernetes/scheduler.kubeconfig \
         --profiling=false \
         --terminated-pod-gc-threshold=500 \
-        --secure-port=0 \
+        --port=0 \
```

Comment:
Do we need to specify `--port` flag? I can't seem to find it in the docs for the `kube-controller-manager`.

Reply:
Specifying `port=0` effectively disables the controller-manager's HTTP server. The port is defined [here](https://github.com/kubernetes/kubernetes/blob/v1.21.2/cmd/kube-controller-manager/app/options/options.go#L235).